src/Catalog/Helpers/FeedPackageDeprecationInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    public sealed class FeedPackageDeprecationInfo
```

Comment:
Why don't we just reuse `PackageDeprecationItem`? Having this additional class doesn't really provide much IMO.

Reply:
Yep! Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/542/commits/b163c15e78dfd6b998ac885c806d36c6c3899c74